client/my-sites/email/titan-set-up-thank-you/index.tsx

```diff
@@ -1,10 +1,7 @@
 import { useTranslate } from 'i18n-calypso';
 import ThankYouV2 from 'calypso/components/thank-you-v2';
-import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
```

Comment:
@billrobbins Looks like the branch needs to be rebased. These changes in `email/titan-set-up-thank-you/index.tsx` have already been merged: https://github.com/Automattic/wp-calypso/pull/88390/files#diff-4de648f48d7bc4d2c6c589e290223d57cea6d434906113c36e52cc88b2877c0f

Reply:
@billrobbins Following up to make sure this ping did not fall through the cracks :)

Reply:
Sorry @wongasy.  I'm getting this fixed up today.  Thanks for the ping 👍